boards/norik/octopus_io_board/Kconfig.defconfig

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2024 Norik Systems
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_OCTOPUS_IO_BOARD
+
```

Comment:
Apply other comments to second commit too

Reply:
Fixed